### DISCUSSION TITLE
New block elements are not saved

### DISCUSSION THREAD

**Discussion by timoRrooaarr** [pimcore/pimcore#10063] [Q&A / Support]
Hi,
in pimcore x we have a block element to be able to add areabricks. this works fine, as long as e new added block entry is first saved, then reoad the whole document and then choose the areabrick, save again, reload again. Then the content e.g. filled in in a wysiwyg editable is saved correct.

Not saving manually betweeen all the steps, let pimcore show "succesfully saved and published", but afert reloading or opening the document, the contents are not there an a emtpy block element wihtou any chossen areabrick is there.

The post parameters in Pimcore show, that the new entry content is named to a id myblock:1.mycontent:1000000. In the database, we can find these entries. But instead of 1000000 this should be the number of the next entry in this block, no?

Any known issues on that problem?
Timo

**Comment by fashxp** [comment]
is that reproducible on the public demo?

**Reply by timoRrooaarr** [reply]
no, in the demo there area also headings (e.g. in News) named with headeing1000001 on new entries, but the editables are named directly correct with the next iterating count.

In my case I could solve the problem, by adding a reload:true to the areablock in the block. With this, the "reordering" seems to fix the wrong nameing and the editable can be saved correct without the need of a manual save and refresh.

I a reload:true mandatory for using areablocks in blocks?

**Reply by fashxp** [reply]
should be not. 
maybe you can investigate a bit further and create a ticket or PR to fix. thx.

**Reply by brusch** [reply]
Maybe caused by #9490 ? 
Are you on latest v10.1.0?

**Reply by timoRrooaarr** [reply]
Hi, sorry for the late reply - holidays ;-)
I am on the 10.1.0 and can not see any conflicts based on duplicate names for areabricks and editables
